.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Probably better to add the file to `.gitignore`. I know it may be useful when using vscode, but I think it's generally better to avoid any IDE specific files